bin/aws_active_job_sqs

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/aws-activejob-sqs'
+require_relative '../lib/aws/active_job/sqs/cli'
+require_relative '../lib/aws/active_job/sqs/poller'
+
+opts = Aws::ActiveJob::SQS::CLI.parse_args(ARGV)
+
+Aws::ActiveJob::SQS::CLI.boot_rails(opts) if opts[:boot_rails]
+
+require File.join(Dir.pwd, opts[:require]) if opts[:require]
```

Comment:
Not sure if we should have a require option but if we do, maybe look at load path instead. I think it should just be passed to ruby's -I and -r somehow when invoking this?

Reply:
This is the mechanism that shoryken uses to get around not booting rails (they take a -r/--require and then require it).

I'm not quite sure how to pass these to ruby -l/-r since this is a script and ruby is run automatically to execute it.

Reply:
Is require and boot_rails mutually exclusive? require may only matter if you use boot rails.

Reply:
I think they are you'd generally only set one but theres nothing that prevents them from being combined.